sorbet/rbi/sorbet-typed/lib/rainbow/all/rainbow.rbi

```diff
@@ -30,7 +30,7 @@ module Rainbow
       sig { returns(Integer) }
       attr_reader :num
 
-      sig { params(ground: Symbol, num: Integer).returns(Indexed) }
+      sig { params(ground: Symbol, num: Integer).void }
```

Comment:
This is my first time making changes because of sorbet. Is this the right thing to do here? @ivoanjo?

Reply:
I'm... not entirely sure. Did you do this change manually, or was it a result of a sorbet autofix? Since this is provided by sorbet, I don't think we're supposed to maintain it manually... 🤔 

I took a slightly different approach an in https://github.com/DataDog/dd-trace-rb/pull/1679 I just re-ran `bundle exec srb init` and sorbet marked this file as `typed: ignore`. (It seems like it's since been fixed upstream https://github.com/sorbet/sorbet-typed/pull/372).

In that PR, I also pinned the Sorbet version so we don't run into these random "oh sorbet updated and now it doesn't like its own rbi" issues again, since this was the second time we got bitten.

So my suggestion would be to just rebase on top of that PR (or master, once merged) instead :)